src/app/app.service.ts

```diff
@@ -31,6 +31,10 @@ export class AppService {
       ".printer should have required property 'zBabystepGCode'": config => (config.printer.zBabystepGCode = 'M290 Z'),
       ".plugins should have required property 'tpLinkSmartPlug'": config =>
         (config.plugins.tpLinkSmartPlug = { enabled: true, smartPlugIP: '127.0.0.1' }),
+      ".plugins should have required property 'tasmota'": config =>
+        (config.plugins.tasmota = { enabled: true, ip: '127.0.0.1', idx: null }),
+      ".plugins should have required property 'tasmotaMqtt'": config =>
+        (config.plugins.tasmotaMqtt = { enabled: true, topic: 'topic', relayN: null }),
```

Comment:
I must have missed that on the Tp-Link SmartPlug PR, the default value for enabled should be `false` here. Would be great if you could also change that for the Line 33.